tbx/static_src/sass/base/_typography.scss

```diff
@@ -42,6 +42,7 @@ h6 {
 .heading {
     @include heading-text();
     font-weight: $weight--semibold;
+    letter-spacing: -1px;
```

Comment:
here, we apply new letter spacing only for components using `heading` class. we would probably want to apply it to any heading tag by moving the rule to the `heading-text` mixin